package-lock.json

```diff
@@ -1,7085 +1,8 @@
 {
   "name": "feder",
   "version": "0.0.2",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
Zaktualuzuj proszę `npm` / `Node`, bo obecnie dokonujesz downgrade `package-lock`, więc testy nie przejdą. W dodatku przyjdzie `renovate-bot` i nam to znowu zaktualizuje. Ewentualnie pomiń ten plik, skoro nie zmieniałeś nic w `package.json`